ns/init_linux.go

```diff
@@ -35,9 +45,21 @@ func SetNamespace() error {
 
 // ParseHandlerInt transforms the namespace handler into a integer
 func ParseHandlerInt() int {
-	return int(initNs)
+	return int(GetHandler())
+}
+
+// GetHandler returns the namespace handler
+func GetHandler() netns.NsHandle {
```

Comment:
Seems like this doesn't need to be exposed as this is only used internal to the package